homeassistant/components/danfoss_air/__init__.py

```diff
@@ -0,0 +1,83 @@
+"""
+Support for Danfoss Air HRV.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/danfoss_air/
+"""
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.const import CONF_HOST
+from homeassistant.helpers import discovery
+import homeassistant.helpers.config_validation as cv
+from homeassistant.util import Throttle
+
+REQUIREMENTS = ['pydanfossair==0.0.6']
+
+_LOGGER = logging.getLogger(__name__)
+
+DANFOSS_AIR_PLATFORMS = ['sensor', 'binary_sensor']
+DOMAIN = 'danfoss_air'
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=60)
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Required(CONF_HOST): cv.string,
+    }),
+}, extra=vol.ALLOW_EXTRA)
+
+
+def setup(hass, config):
+    """Set up the Danfoss Air component."""
+    conf = config[DOMAIN]
+
+    hass.data[DOMAIN] = DanfossAir(conf[CONF_HOST])
+
+    for platform in DANFOSS_AIR_PLATFORMS:
+        discovery.load_platform(hass, platform, DOMAIN, {}, config)
+
+    return True
+
+
+class DanfossAir:
+    """Handle all communication with Danfoss Air CCM unit."""
+
+    def __init__(self, host):
+        """Initialize the Danfoss Air CCM connection."""
+        self._data = {}
+
+        from pydanfossair.danfossclient import DanfossClient
+
+        self._client = DanfossClient(host)
+
+    def get_value(self, item):
+        """Get value for sensor."""
+        if item in self._data:
+            return self._data[item]
```

Comment:
```py
def get_value(self, item):
    return self._data.get(item)
```

Reply:
I was not aware of the difference between [] and .get(). Python is not one of my primary programming languages :-) It will however make the code a bit more simple. I will correct this in new PR. Thanks for the hint.